app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -23,5 +23,14 @@ def logout_initiated(user_uuid:, unique_session_id:, success:)
         success: success,
       )
     end
+
+    # Tracks when the user has attempted to enroll the MFA method TOTP to their account
+    # @param [Integer] enabled_mfa_methods_count number of registered mfa methods for the user
```

Comment:
this comment is a lie
```suggestion
    # @param [Boolean]success
```

Reply:
Ha!  Sorry about that! 😅 😉  Taken care of.